.circleci/config.yml

```diff
@@ -118,14 +118,39 @@ jobs:
     - store_artifacts:
         path: /tmp/circleci-test-results
   sharness:
-    executor: golang
+    resource_class: 2xlarge
```

Comment:
Is this still required or is it overkill we were using while trying to debug CI?

Reply:
Not required. Removed.

Reply:
Not required, but parallelism cannot go above -j 3 without it (tests become flaky, like dag-import...), according to my experiments. Removed 2xlarge for now.